case-lib/hijack.sh

```diff
@@ -105,6 +105,11 @@ function func_exit_handler()
         done
     fi
 
+    # Currently devres debug log is enabled only for kmod test.
+    # When the test case is failed, devres debug level may not be clear.
+    # So exit handler should reset log level.
+    disable_devres_debug_log
```

Comment:
I would have put this earlier and/or next to some other kernel related code like the kernel checkpoint stuff but good enough. You confirm none of the commands above will cause more verbose logs, correct? These verbose logs happen only on driver (un)loading?

Reply:
I tested with playback test with devres log enabled. No devres log. But devres may be called in any time. Yes, kmod tests have most devres verbose log.

Reply:
let's use traces as suggested by Andy Shevchenko here; https://github.com/thesofproject/linux/issues/3008#issuecomment-871444928

I also wonder if we might enable more traces that just devres.

Reply:
Good to know we can dump ftrace dump on oops. 
And also looking at traces, what to enable is not so intuitive.

Reply:
> let's use traces as suggested by Andy Shevchenko here; thesofproject/linux#3008 (comment)

Does that mean we should not merge this? Or can we merge it and change it or add to it later?

Sorry I don't have the time to take a crash course on devres  events right now.

Reply:
not merge and rethink in light of new information.